web/modules/custom/rdf_entity/src/RdfPermissions.php

```diff
@@ -41,19 +41,25 @@ public function rdfTypePermissions() {
    */
   protected function buildPermissions(RdfEntityType $type) {
     $type_id = $type->id();
-    $type_params = array('%type_name' => $type->label());
+    $type_params = ['%type_name' => $type->label()];
 
-    return array(
-      "create $type_id rdf entity" => array(
+    return [
+      "create $type_id rdf entity" => [
```

Comment:
This is a contributed module, we shouldn't make changes in it to benefit our project.

There are initiatives ongoing for reworking the permissions in Drupal, see for example https://www.drupal.org/node/2744683#comment-11274521

If we need machine based interaction with permissions then the best thing we can do is to abstract it in classes, similar to how I've done it in OG. Relying on some dumb fixed string format is a very poor way of dealing with this. See https://github.com/amitaibu/og/issues/222

Here's how this could look: https://github.com/Gizra/og/blob/8.x-1.x/src/Permission.php

I already wrote all of this, so adapting it to our use case is not going to be too difficult.